package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "root",
```

Comment:
shouldn't we add a dependabot entry for this new package file?

Reply:
Yeah right

Reply:
Done